### ISSUE TITLE
Lr scheduler resets when resuming model

### ISSUE THREAD

**Issue by Noahprog** [chnsh/DCRNN_PyTorch#5]
Hi, many thanks for converting the tensorflow implementation of DCRNN to pytorch, it helped me a lot. 

I noticed that when resuming a model, the learning-rate scheduling is being reset. Any idea why this happens? I uploaded a picture of a test case below.
If you need any further details, let me know.

<img width="977" alt="Screenshot 2020-05-25 at 17 23 26" src="https://user-images.githubusercontent.com/31852504/82826148-da2a7300-9eac-11ea-8621-89561960df4a.png">

**Comment by chnsh** [comment]
That's a good find - can you submit a PR to fix it? @Noahprog
